psydac/api/fem.py

```diff
@@ -438,7 +449,7 @@ def assemble(self, *, reset=True, **kwargs):
                     assert len(self.grid) == 1
                     if not v.coeffs.ghost_regions_in_sync:
                         v.coeffs.update_ghost_regions()
-                    basis_v  = BasisValues(v.space, nderiv = self.max_nderiv, trial=True, grid=self.grid[0])
+                    basis_v  = BasisValues(v.space, nderiv = self.max_nderiv, trial=True,  grid=self.grid[0])
```

Comment:
There is an extra space before `grid=` and before the assignment

Reply:
Fixed in 6b86a05